restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/DaoUtils.java

```diff
@@ -199,4 +217,12 @@ public static String writeApplicationKind(Application.Kind kind) {
         }
     }
 
+    public static String writeGeolocationType(Geolocation.GeolocationType geolocationType) {
```

Comment:
@FerUy seems that this method never gets called, if that is true and there are no plans to use it later, better to remove it